src/editor/components/components/AddLayerPanel/createLayerFunctions.js

```diff
@@ -61,7 +62,7 @@ export function createStreetmixStreet(position, streetmixURL, hideBuildings) {
   // position the street further from the current one so as not to overlap each other
   if (streetmixURL && streetmixURL !== '') {
     const definition = {
-      id: streetmixURL,
+      id: createUniqueId(),
```

Comment:
why this change?

Reply:
streetmixURL is not unique, dragging and dropping the same street preset from the add layer panel will cause id collision errors